### PR TITLE
Add PHP-CS-Fixer and fix CS

### PR DESCRIPTION
**Pull request by meyerbaptiste**

### CODE REVIEW

**Review by GregoireHebert — APPROVED** [review]
Maybe adding some phpDoc here :
`spec/CoopTilleuls/Bundle/YmlpBundle/Ymlp/YmlpClientSpec.php` ?

**Review by toofff — APPROVED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+root = true
+[*]
+indent_style = space
+indent_size = 4
```
Comment (meyerbaptiste) [inline .editorconfig]: ? 🤔

File: composer.json [hunk 2]
```diff
@@ -14,13 +14,14 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/config": "~2.3||~3.0",
-        "symfony/dependency-injection": "~2.3||~3.0",
-        "symfony/http-kernel": "~2.3||~3.0",
-        "guzzlehttp/guzzle": "~4.0"
+        "symfony/config": "^2.3 || ^3.0",
+        "symfony/dependency-injection": "^2.3 || ^3.0",
+        "symfony/http-kernel": "^2.3 || ^3.0",
```
Comment (Simperfit) [inline composer.json]: why spacing this ?

File: .editorconfig (hunk 1 above)
Comment (toofff) [inline .editorconfig]: For the file `.editorconfig`

File: composer.json (hunk 2 above)
Comment (meyerbaptiste) [inline composer.json]: They use spaces in the documentation of Composer and I'm maniacal.

Comment (dunglas) [inline composer.json]: +1 for spaces, it's easier to read.

**Review by dunglas — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
 matrix:
+  include:
+    - php: 5.4
+      env: deps='low' lint=1
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+    - php: hhvm
   allow_failures:
```
Comment (dunglas) [inline .travis.yml]: Why not removing this line as HHVM tests are green?

**Pull request merged**